luigi_pipeline/README.md

```diff
@@ -0,0 +1,31 @@
+
+# Luigi Work in Progress
+
+## Local
+### Install
+```
+$ pip install -r requirements.txt
+```
+
+### Tests
+```
+$ pip install nose
```

Comment:
It's just the one I've always used, but I believe you should be able to run it with others.

Reply:
since you're only importing `unittest` in your tests, I would remove the nose dependency unless there's a major reason to use it. Want to avoid some tests casually becoming dependent on it.

Reply:
Yeah I can change it to whatever if it's a small change, but just wanted to point out that nose knows.